core/src/main/java/io/crate/types/IntegerType.java

```diff
@@ -95,4 +95,9 @@ public void writeValueTo(StreamOutput out, Object v) throws IOException {
     public DataType<?> create() {
         return INSTANCE;
     }
+
+    @Override
+    public int fixedSize() {
```

Comment:
add a comment that the actual size is not 8 but is padded to 8, same for other types please